packages/stores/src/queries/pool-incentives/incentivized-pools.ts

```diff
@@ -118,6 +118,52 @@ export class ObservableQueryIncentivizedPools extends ObservableChainQuery<Incen
     }
   );
 
+  readonly computeAPYWithExternalIncentives = computedFn(
+    (
+      poolId: string,
+      duration: Duration,
+      priceStore: IPriceStore,
+      fiatCurrency: FiatCurrency,
+      externalGauges: ExternalGauge[]
+    ): RatePretty => {
+      const apy = this.computeAPY(poolId, duration, priceStore, fiatCurrency);
+
+      if (!externalGauges.length) {
+        return apy;
+      }
+
+      const externalGauge = externalGauges.find((externalGauge) => {
+        return (
+          duration.asMilliseconds() === externalGauge.duration.asMilliseconds()
+        );
+      });
+
+      if (!externalGauge?.rewardAmount) {
+        return apy;
+      }
+
+      const mintDenom = this.queryMintParmas.mintDenom;
+
+      if (!mintDenom) {
+        return apy;
+      }
+
+      const chainInfo = this.chainGetter.getChain(this.chainId);
+
+      const mintCurrency = chainInfo.findCurrency(mintDenom);
+
+      if (!mintCurrency) {
+        return apy;
+      }
+
+      const amount = externalGauge.rewardAmount.moveDecimalPointLeft(
+        mintCurrency.coinDecimals
```

Comment:
Diving the value by `1_000_000` gives me the value I expect, but I figured hard-coding doesn't make sense. So I took this from `computeAPYForDuration`. Let me know I should be getting this _value_ elsewhere.